cla_public/apps/contact/constants.py

```diff
@@ -7,3 +7,7 @@
 DAY_TODAY = "today"
 DAY_SPECIFIC = "specific_day"
 DAY_CHOICES = ((DAY_TODAY, _("Call today")), (DAY_SPECIFIC, _("Call on another day")))
+SELECT_OPTION_DEFAULT = [("", "-- Please select --")]
```

Comment:
The labels need to be wrapped in _() so it can be available for translation for welsh

Reply:
Nice spot, thanks. I've added this change.

Reply:
@psweeting1 I think you also need to to do if for the error messages too:
```SELECT_OPTION_DEFAULT = [("", _("-- Please select --"))]
TIME_TODAY_VALIDATION_ERROR = u"Select what time you want to be called today"
DAY_SPECIFIC_VALIDATION_ERROR = u"Select which day you want to be called"
TIME_SPECIFIC_VALIDATION_ERROR = u"Select what time you want to be called"```

Reply:
By making them _() they fail unit tests for rendering the strings correctly. They are passed into _() via the message argument in fields.py.